person.rb

```diff
@@ -0,0 +1,40 @@
+require_relative './student'
+require_relative './teacher'
+
+class Person
+  extend Student
+  extend Teacher
+
+  def initialize(age, name: 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age, :parent_permission
```

Comment:
- [ ] Kindly consider removing parent-permission from the attr-accessor since it is not supposed to be read or altered. It is supposed to be a private property that defines whether a teenager can use service or not.